content/en/synthetics/settings/_index.md

```diff
@@ -87,20 +87,20 @@ To generate and use a TOTP in your tests, create a global variable where you ent
 
 {{< /tabs >}}
 
-### Permissions
```

Comment:
This section should actually stay here. The restrict access section is a subsection for that one.